app/assets/fonts/generator_config.txt

```diff
@@ -1,5 +0,0 @@
-# Font Squirrel Font-face Generator Configuration File
```

Comment:
There are a lot of files in the `app/assets/fonts` folder that aren't fonts.  This causes problems on deploys.